### ISSUE TITLE
An indication of the realtime direction of the shadow as cast by the sun.

### ISSUE THREAD

**Issue by jandevaan** [organicmaps/organicmaps#5951]
**Is your feature request related to a problem? Please describe.**
The indication would just help to match the phone map orientation to the real world, which helps to concentrate on the trail rather than your phone. It helps to be able to remember that the next fork in the road the correct path should be 45 degrees to the left of your own shadow. It can help you to learn the sun for orientation, with/without your phone. 
I am aware that phones are somewhat direction sensitive, but I find this unreliable. During daytime and with clear weather, the sun is way more accurate. 
 
**Describe the solution you would like**
I would like to see a little person where I stand on the map, which casts a shadow taking the sun's location and my position into account.
I am aware it only works in daytime when it is not too cloudy. It should not shown anything when the sun is below the horizon. 

**Describe alternatives you have considered**
What I do now is trying to orient based on the current trail direction, if there is one.
I don't think anybody else has this feature. 
An alternative design would be to render the sun/shadow on top of the compass. That is likely easier but it is a bit more abstract for casual users to notice. 

**Additional context**
Nothing that comes to mind.

**Comment by janusbenissa** [comment]
> I am aware it only works in daytime when it is not too cloudy. It should not shown anything when the sun is below the horizon.

And if you are no perpendicular under the sun ;-)

**Comment by jandevaan** [comment]
Yeah, that's an edge case to consider too. But I am mostly glad that someone reads my post. The point is mostly that I think users would not mind that this feature does not work 100% of the time...
